Foundation/Data.swift

```diff
@@ -1553,7 +1558,7 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     /// - parameter buffer: The replacement bytes.
     @inline(__always)
     public mutating func replaceSubrange<SourceType>(_ subrange: Range<Index>, with buffer: UnsafeBufferPointer<SourceType>) {
-        guard !buffer.isEmpty else { return }
```

Comment:
Please don’t revert the changes in https://github.com/apple/swift-corelibs-foundation/pull/1485.

Reply:
@saiHemak what happened here? I thought you were taking the latest version of Data from the overlay?